tools/cli/flags.go

```diff
@@ -101,6 +101,8 @@ const (
 	FlagPrintFullyDetailWithAlias   = FlagPrintFullyDetail + ", pf"
 	FlagPrintRawTime                = "print_raw_time"
```

Comment:
Is it still being used by other command? If not we may delete it.

Reply:
used by others